external-crates/move/crates/move-package-alt/src/graph/mod.rs

```diff
@@ -155,96 +134,3 @@ impl<F: MoveFlavor> PackageGraph<F> {
         }
     }
 }
-
-#[cfg(test)]
```

Comment:
These tests had accidentally been duplicated to package_info during a merge they are there now